local-cli/runIOS/runIOS.js

```diff
@@ -129,7 +133,15 @@ function buildProject(xcodeProject, udid, scheme) {
     '-derivedDataPath', 'build',
   ];
   console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
-  child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'inherit'});
+  var buildProcess = child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'pipe'});
+  var buildOutput = buildProcess.stdout.toString();
+  console.log(buildOutput);
```

Comment:
I think the spawn approach is more correct here. We're seeing issues with console output being hidden already on travis (e.g. your build hits a timeout and gets killed, this console.log will never get executed). 

Given that in the default configuration for RN, the app name is identical to the scheme, I think we should try to get it right instead of a hacking a fix in.